core/store/eth_client.go

```diff
@@ -167,3 +169,33 @@ var emptyHash = common.Hash{}
 func (txr *TxReceipt) Unconfirmed() bool {
 	return txr.Hash == emptyHash || txr.BlockNumber == nil
 }
+
+// TxReceiptStatus encapsulates the different return values available
+// for an ethereum transaction receipt as defined by
+// http://eips.ethereum.org/EIPS/eip-658
+type TxReceiptStatus string
```

Comment:
This looks like a number, yet we are storing it as a string?

Reply:
We're actually going to end up storing stings, so I think this is appropriate for what we'll send in the future, I'll update this in my next PR.